### PR TITLE
ephemeralpg 1.9 (new formula)

### PR DESCRIPTION
**Pull request by LinusU**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

**Comment by eradman** [comment]
`pg_tmp` and `man pg_tmp` work perfectly for me

**Comment by LinusU** [comment]
Nice :raised_hands: thanks for the test

File: Library/Formula/ephemeralpg.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Ephemeralpg < Formula
+  desc "Run tests on an isolated, temporary Postgres database"
+  homepage "http://ephemeralpg.org"
+  url "http://ephemeralpg.org/code/ephemeralpg-1.9.tar.gz"
+  mirror "https://bitbucket.org/eradman/ephemeralpg/get/ephemeralpg-1.9.tar.gz"
+  sha256 "3caf06f2be5d9f206f3c1174cc0c44cc359357fc7d41da026f858e01ef192792"
+
+  depends_on "postgresql"
```
Comment (DomT4) [inline Library/Formula/ephemeralpg.rb]: This can probably be: `depends_on :postgresql`

**Comment by LinusU** [comment]
@DomT4 @dunn Updated :raised_hands:

**Comment by dunn** [comment]
Accidentally merged without bottles, fixed that in b246e0eec152a561847a3cb89bc0394c1d8eac48.

**Comment by LinusU** [comment]
Thanks!

**Pull request closed without merging**